dropwizard-client/src/main/java/io/dropwizard/client/ConfiguredCloseableHttpClient.java

```diff
@@ -0,0 +1,22 @@
+package io.dropwizard.client;
+
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.impl.client.CloseableHttpClient;
+
+/* package */ class ConfiguredCloseableHttpClient {
+    private final CloseableHttpClient closeableHttpClient;
+    private final RequestConfig defaultRequestConfig;
+
+    /* package */ ConfiguredCloseableHttpClient(CloseableHttpClient closeableHttpClient, RequestConfig defaultRequestConfig) {
+        this.closeableHttpClient = closeableHttpClient;
+        this.defaultRequestConfig = defaultRequestConfig;
+    }
+
+    public RequestConfig getDefaultRequestConfig() {
+        return defaultRequestConfig;
+    }
+
+    public CloseableHttpClient getClient() {
+        return closeableHttpClient;
+    }
+}
```

Comment:
Trivial nitpick.

Please, add a newline in the end of the added files. It'a standard for text files on *nix systems.
[Reference] (http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline)